src/actions/noting/reset-note-barriers.js

```diff
@@ -0,0 +1,35 @@
+var isVFocus = require('../../utils/vfocus/is-vfocus');
+var isVText = require('../../utils/vfocus/is-vtext');
+var findAllNotes = require('../../utils/noting/find-all-notes');
+var createNoteBarrier = require('../../utils/create-note-barrier');
+var isNotWithinNote = require('../../utils/noting/is-not-within-note');
+var isNotEmpty = require('../../utils/vfocus/is-not-empty');
+var errorHandle = require('../../utils/error-handle');
+
+module.exports = function resetNoteBarriers(focus) {
+
+  if (!isVFocus(focus)) {
+    errorHandle('Only a valid VFocus element can be passed to resetNoteBarriers, you passed: ', focus);
+  }
+
+  //remove any note barriers that exist
+  focus.filter(isVText).forEach(focus => {
+    focus.vNode.text = focus.vNode.text.replace(/\u200B/g, '');
+  });
+
+  //add new note barriers
+  findAllNotes(focus).forEach(noteSegments => {
+    //first note
+    noteSegments[0].next().insertBefore(createNoteBarrier());
```

Comment:
A comment why we insert the first note barrier inside the note and the other one after the note would be good. Think there might've been one in the original code that you can reuse.

Important since it's not apparent and changing this will cause bugs in Chrome.